1-exercises/B-array-some/exercise.js

```diff
@@ -11,6 +11,11 @@ let pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
 // If there is a null value in the array exit the program with the error code
 // https://nodejs.org/api/process.html#process_process_exit_code
 // process.exit(1);
+function checkNullNumbers(array) {
+  return array === null;
+}
+const containNull = pairsByIndex.some(checkNullNumbers);
```

Comment:
good job think about if you use if/else makes your code shrter and easier to read